src/components.rs

```diff
@@ -15,7 +15,7 @@ pub fn base(content: Markup) -> Markup {
                 meta name="viewport" content="width=device-width,initial-scale=1.0" {}
             }
             body class="bg-slate-100 dark:bg-neutral-900" {
-                div id="app-container" class="lg:px-64" {
```

Comment:
please add padding for xl (or maybe 2xl) screens. this is gonna look really weird on an ultrawide monitor without it

Reply:
Okie dokie c6ba07f62c7f603c23f565ca3aa6ebd16e9e4247
[untitled.webm](https://github.com/user-attachments/assets/97e36ee3-4355-4de1-89d5-9b7e4cbe9c34)

Reply:
Perfect!